modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

```diff
@@ -12,6 +12,15 @@ WARNING: `CHE_INFRA_KUBERNETES_NAMESPACE` and `CHE_INFRA_OPENSHIFT_PROJECT` are
 
 WARNING: By default, only one workspace in the same {orch-namespace} can be running at one time. See xref:running-more-than-one-workspace-at-a-time.adoc[].
 
+[WARNING]
+====
+By {kubernetes} definition, the maximum length of namespace name is limited to 63 characters, including the evaluated placeholders.
+
+On OpenShift with multi-host server exposure strategy, the length is further limited to 49 characters.
+
+Be aware that `_<userid>_` placeholder is evaluated into 36 character long UUID string.
```

Comment:
Be aware that the \`\_\<userid\>\_\` placeholder is evaluated into a 36 character long UUID string.

Reply:
thanks, updated